lib/media/streaming_engine.js

```diff
@@ -2688,7 +2695,7 @@ shaka.media.StreamingEngine = class {
     this.destroyer_.ensureNotDestroyed();
     shaka.log.v2(logPrefix, 'appended media segment');
   }
-
+  
```

Comment:
There's a couple of unnecessary spaces and line breaks, let's remove them as they bring no value to the PR.